pwlwp/chat.py

```diff
@@ -0,0 +1,38 @@
+import openai
+import os
+
+class Chat:
+
+	question = ""
+	model = "gpt-4"
+	timestr = ""
+	current_prompt = 0
+
+	def __new__(cls, *args, **kwargs):
+		return super().__new__(cls)
+
+	def __init__(self, model, question, timestr):
+		openai.api_key_path = "openai_key"
+		self.question = question
+		self.model = model
+		self.timestr = timestr
+		os.mkdir("gpt_chat/" + timestr)
+
+	def calL_gpt(self) -> str:
+
+		chat_completion = openai.ChatCompletion.create(model=self.model,
+													   messages=[{"role": "user", "content": self.question}])
+
+		f = open("gpt_chat/" + self.timestr + "/" + str(self.current_prompt) + "_input.txt", "w")
```

Comment:
At some point someone will go through and change these to `with` statements and check the encoding.  I see one example in the code that looks like this:
```pythons
with open(file_name, encoding="utf-8") as file:
```